test/e2e-olm/utils.go

```diff
@@ -366,3 +349,42 @@ func getCertificateContents(name, uuid string) []byte {
 	filename := fmt.Sprintf("%s/%s/%s", exampleSecretsPath, uuid, name)
 	return utils.GetFileContents(filename)
 }
+
+func getProjectRootPath(projectRootDir string) string {
+	fmt.Println("### getProjectRootPath is running")
+	cwd, err := os.Getwd()
+	cwdOrig := cwd
+	if err != nil {
+		panic(err)
+	}
+	for {
+		if strings.HasSuffix(cwd, "/"+projectRootDir) {
+			return cwd
+		}
+		lastSlashIndex := strings.LastIndex(cwd, "/")
+		if lastSlashIndex == -1 {
+			panic(cwdOrig + " did not contain /" + projectRootDir)
+		}
+		cwd = cwd[0:lastSlashIndex]
+		fmt.Printf("cwd %v\n", cwd)
+	}
+}
+
+func waitForDeleteObject(t *testing.T, client client.Client, key types.NamespacedName, obj runtime.Object, retryInterval, timout time.Duration) error {
```

Comment:
- Why does this function need a `client.Client` parameter and not using it?

Reply:
Fixed